docs/template_plugin/tests/functional/subgraph_reference/preprocess.cpp

```diff
@@ -505,6 +529,152 @@ static RefPreprocessParams resize_and_convert_layout() {
     return res;
 }
 
+static RefPreprocessParams convert_color_nv12_to_bgr_two_planes() {
+    RefPreprocessParams res("convert_color_nv12_to_bgr_two_planes");
+    res.abs_threshold = 2.f; // Allow small color conversion deviations
+    res.rel_threshold = 1.f; // Ignore relative pixel values comparison (100%)
```

Comment:
Is it ok to have so big threshold?

Reply:
For color conversion, from requirements perspective it is important to have result close to reference, and it is ok if values are same +- small deviation (2.f in my case). So, if some plugin will give (254, 0, 0) instead of (255, 0, 0) - it is fine.
There is no reason to have relative comparison for color conversion tests, as (1,1,1) comparing with (0,0,0) will give relative error = (1-0)/1=100%, but for (254,0,0) vs (255,0,0) relative error will be (255-254)/255 < 0.5%. But error is the same (1 level of 255). Thus I decided to turn relative comparison off just to set 'threshold' to 100% (1.f)